meteor-app/imports/startup/client/routes.js

```diff
@@ -2,16 +2,25 @@ import React from 'react'
 import { BrowserRouter as Router, Route, Switch } from 'react-router-dom'
 import isIframe from '/imports/helpers/isIframe'
 import Attendance from '/imports/ui/layouts/attendance'
+import Parts from '/imports/ui/layouts/parts'
+import Assessment from '/imports/ui/layouts/assessment'
```

Comment:
I think I want change these names.
Let's rename `Parts` to be `Ordering`, and `Assessment` to be `Servicing` (the focus has shifted a little on this one). Let's rename files and component names to suit.
It also allows us to use `Parts` and `Services` for the database collection names respectively, without fear of it colliding with a layout page